README.md

```diff
@@ -78,4 +79,11 @@ make run
 ## Run all commands at once
 ```
 make all
-```
\ No newline at end of file
+```
+
+# Docker Container
+
+To build the container, make sure that the Dockerfile is present and run `docker build -t common-backend:latest .` 
+
+To run the application from the docker container, run `docker run common-backend ARG` where `ARG` can be `db`, `run, test`, `shell` or `runserver`.  
```

Comment:
I think this should be `docker run ARG` since we have `python manage.py` as endpoint.

What do you say?

Reply:
common-backend is the name of the docker container. ARG is the argument we pass, such as run or test. I think it has to be present.